src/components/LikeCoinButtonWidgetV2/LikeCoinButtonWidgetV2.vue

```diff
@@ -135,6 +147,18 @@ export default {
       type: Boolean,
       default: false,
     },
+    tempLink: {
+      type: String,
+      default: '',
+    },
+    tempCTA: {
+      type: String,
+      default: '',
+    },
+    tempNFTPrice: {
+      type: Number,
+      default: -1,
+    },
```

Comment:
Don't name these `temp` in code, we will use a `temp` feature branch to denote these change are temp